src/pages/sighting/Sighting.jsx

```diff
@@ -1,270 +1,27 @@
-import React, { useMemo, useState } from 'react';
-import { useParams, useHistory } from 'react-router-dom';
-import { useIntl, FormattedMessage } from 'react-intl';
-import { get } from 'lodash-es';
-import Tabs from '@material-ui/core/Tabs';
-import Tab from '@material-ui/core/Tab';
+import React from 'react';
+import { useParams } from 'react-router-dom';
 
-// import SpeciesIcon from '@material-ui/icons/Category';
-// import RegionIcon from '@material-ui/icons/MyLocation';
-// import ContextIcon from '@material-ui/icons/NaturePeople';
-// import SubmitterIcon from '@material-ui/icons/Person';
-
-import defaultSightingSrc from '../../assets/defaultSighting.png';
-import MainColumn from '../../components/MainColumn';
-import Link from '../../components/Link';
-import Text from '../../components/Text';
-import LoadingScreen from '../../components/LoadingScreen';
-import SadScreen from '../../components/SadScreen';
-import Button from '../../components/Button';
-import MoreMenu from '../../components/MoreMenu';
-import ConfirmDelete from '../../components/ConfirmDelete';
-import EntityHeaderNew from '../../components/EntityHeaderNew';
-import useAssetGroupSighting from '../../models/assetGroup/useAssetGroupSighting';
+import SightingCore from './SightingCore';
```

Comment:
This now seems like  a wrapper for SightingCore. Why is this file necessary?

Reply:
Missed this one too. The structure is now different - Sighting.jsx and AssetGroupSighting.jsx do the requests and supply data to the SightingCore component which renders everything